src/models.py

```diff
@@ -237,13 +237,17 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
             d_word = word_embs.size()[1]
             log.info("\tUsing pre-trained word embeddings: %s",
                      str(word_embs.size()))
+        elif args.word_embs=="scratch":
+            log.info("\tScratch Word embs!")
+            d_word=args.d_word
+            word_embs= nn.Embedding(n_token_vocab, d_word).weight
         else:
```

Comment:
What does this clause do? It's a serious problem to have two different options that both claim to be learning embeddings from scratch.

Reply:
Fixed